src/auro-accordion.js

```diff
@@ -94,6 +95,10 @@ export class AuroAccordion extends LitElement {
       variant: {
         type: String,
         reflect: true
+      },
+      disabled: {
+        type: Boolean,
+        reflect: true
```

Comment:
You should also be able to disable an entire accordion group without having to do the accordions individually.